src/xml_processing/tests/pmids_by_mods

```diff
@@ -1,5 +1,5 @@
 10206683	1	RGD
-10022914	3	RGD, FB, WB
```

Comment:
I think this file shouldn't change.  It gets changed by the functional tests, but the changes shouldn't get committed.  Maybe we should add to the .gitignore ?